apps/allocations/app/components/Card/Empty.js

```diff
@@ -24,7 +24,6 @@ Empty.propTypes = {
 
 const EmptyWrapper = styled.div`
   ${unselectable};
-  flex-grow: 1;
```

Comment:
I removed this rule because I used the same Empty card from AddressBook, to keep the same css on both, if it is needed for any reason we should re-add to both apps (and probably the rest of apps when their design is updated)